index.html

```diff
@@ -10,22 +10,48 @@
 </head>
 
 <body>
-    <section>
-        <h1>Awesome Books</h1>
+    <nav>
+        <h2>Awesome Books</h2>
+        <ul>
+            <li><a id="list-link" href="#">List</a></li>
+            <li><a id="add-link" href="#">Add new</a></li>
+            <li><a id ="contact-link" href="#">Contact</a></li>
+        </ul>
+    </nav>
```

Comment:
✅ It has a Navigation bar that navigates great 👍🏻.
✅ When any of the links are clicked, the main area changes to display only one of the three content sections 👍🏻.